app/scripts/models/auth_brokers/fx-fennec-v1.js

```diff
@@ -17,6 +17,7 @@ define(function (require, exports, module) {
 
   var FxFennecV1AuthenticationBroker = FxSyncWebChannelAuthenticationBroker.extend({
     defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
+      browserTransitionsAfterEmailVerification: false,
```

Comment:
This was an error before! Fennec always showed the green checkbox when it should have been showing CAD!